draft-ietf-taps-arch.md

```diff
@@ -160,12 +165,12 @@ The Transport Services architecture maintains this general model of interaction,
 +-----------------------------------------------------+
                           |
 +-----------------------------------------------------+
-|               Transport Services API                |
+|   Transport Services API (resolution + connection)  |
 +-----------------------------------------------------+
                           |
```

Comment:
Also do not like the "(resolution + connection)". What about sending and receiving data? Also looks like the API exposes resolution and connection as separate entities rather than the opposite.

Reply:
Yep, have reverted this, it's clear in contrast with the previous figure.